<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <title>SINGLE CELL MAP</title>
</head>
<body>
  <p>
    Hello!
    <br>Welcome to the single cell map!
    <br>You can open the dataset by clicking on the button to your left.
    <br>After you open the dataset map you can either see the different cell annotations or specific genes by clicking on the buttons on the left side. 
      <br>nCount= number of UMIs
      <br>nFeature= number of genes
      <br>group= the experimental group the cells come from
      <br>percent.mito= percentage of mitochondrial genes detected
<br>
    <br>You can also click on the cluster number to see the marker genes for each cluster.
      <br>avg logFC= average log fold change
      <br>p val adj= adjusted p value; note this may be higher than 0.05 as genes with nominal p value less than 0.05 were kept
      <br>pct.1=percentage of cells expressing the gene in the cluster
      <br>pct.2=percentage of cells expressing the genes in all other clusters
      <br>p-value=nominal p value
<br>
<br>Under the "genes" tab, the top 100 upregulated genes from NAC (based on previous bulk RNA seq) are shown, but you can search for any gene and see its expression level on the cellular map.
<br>
    <br>Finally under the genes tab, you can manually select a subgroup of cells by drawing a square (after clicking on the "square" button on the top left corner) and a violin plot with the gene expression for the selected cells pops up on the right side.

  </p>
</body>
</html>
